app/models/merchant.rb

```diff
@@ -26,4 +26,16 @@ def distinct_cities
     item_orders.distinct.joins(:order).pluck(:city)
   end
 
+  def disable_all_items
+    # Item.where(merchant_id: self.id).update_all(active?: false) #the question mark != creamy
```

Comment:
Heres why you don't label a column with a ? because AR doesn't like it!

Reply:
Isn't that just something?